plugins/modules/postgresql_pg_hba.py

```diff
@@ -1274,11 +1330,16 @@ def main():
         'state',
         'users'
     ]
-    if rules is None:
-        single_rule = dict()
-        for key in rule_keys:
-            single_rule[key] = module.params[key]
-        rules = [single_rule]
+    new_rules = []
```

Comment:
```suggestion
    new_rules = []
```
could we move the below lines to a separate function for logic readability?

Reply:
yes, it does, but I would do this in a later PR